src/middleware/options/ChaosHandlerOptions.ts

```diff
@@ -48,20 +48,30 @@ export class ChaosHandlerOptions implements MiddlewareOptions {
 	 */
 	public chaosPercentage: number;
 
+	/**
+	 * The response body to be returned in the response
+	 *
+	 * @public
+	 */
+	public responseBody: any;
```

Comment:
Just a thought (no action): A blog post about how to use ChaosHandler to test customer integration with this client would be interesting.